frame/tips/src/lib.rs

```diff
@@ -408,6 +410,35 @@ decl_module! {
 			Tips::<T>::remove(hash);
 			Self::payout_tip(hash, tip);
 		}
+
+		/// Remove and slash an already-open tip.
+		///
+		/// May only be called from `T::RejectOrigin`.
+		///
+		/// As a result, API will slash the finder and the deposits are lost.
```

Comment:
```suggestion
		/// As a result, the finder is slashed and the deposits are lost.
```

Who is API? I think this adds ambiguity, use passive.